### ISSUE TITLE
Mapeo gnu/Linux Installer AppImage is broken 

### ISSUE THREAD

**Issue by arky** [digidem/mapeo-desktop#500]
You can not install the Mapeo AppImage on gnu/Linux machines 

```

$ ./Installar_Mapeo_v5.4.0-beta.14_linux.AppImage 
A JavaScript error occurred in the main process
Uncaught Exception:
TypeError [ERR_INVALID_URL]: Invalid URL: /desktop
    at onParseError (internal/url.js:243:9)
    at new URL (internal/url.js:319:5)
    at Object.newBaseUrl (/tmp/.mount_InstalutNKYd/resources/app.asar/node_modules/electron-updater/out/util.js:10:20)
    at new GenericProvider (/tmp/.mount_InstalutNKYd/resources/app.asar/node_modules/electron-updater/out/providers/GenericProvider.js:12:31)
    at Object.createClient (/tmp/.mount_InstalutNKYd/resources/app.asar/node_modules/electron-updater/out/providerFactory.js:43:20)
    at AppImageUpdater.setFeedURL (/tmp/.mount_InstalutNKYd/resources/app.asar/node_modules/electron-updater/out/AppUpdater.js:165:42)
    at new MapeoUpdater (/tmp/.mount_InstalutNKYd/resources/app.asar/src/main/auto-updater.js:31:17)
    at Object.<anonymous> (/tmp/.mount_InstalutNKYd/resources/app.asar/src/main/auto-updater.js:192:25)
    at Module._compile (internal/modules/cjs/loader.js:967:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1004:10)
```

$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.10
Release:	20.10
Codename:	groovy

**Comment by arky** [comment]
The debian package also suffers from the same issue. 

```shell
$ sudo dpkg -i Installar_Mapeo_v5.4.0-beta.14_linux.deb 
[sudo] password for arky: 
Selecting previously unselected package mapeo-desktop.
(Reading database ... 573414 files and directories currently installed.)
Preparing to unpack Installar_Mapeo_v5.4.0-beta.14_linux.deb ...
Unpacking mapeo-desktop (5.4.0-beta.14) ...
Setting up mapeo-desktop (5.4.0-beta.14) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu1) ...
Processing triggers for desktop-file-utils (0.24-1ubuntu4) ...
Processing triggers for mime-support (3.64ubuntu1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...


$ mapeo-desktop 
A JavaScript error occurred in the main process
Uncaught Exception:
TypeError [ERR_INVALID_URL]: Invalid URL: /desktop
    at onParseError (internal/url.js:243:9)
    at new URL (internal/url.js:319:5)
    at Object.newBaseUrl (/opt/Mapeo/resources/app.asar/node_modules/electron-updater/out/util.js:10:20)
    at new GenericProvider (/opt/Mapeo/resources/app.asar/node_modules/electron-updater/out/providers/GenericProvider.js:12:31)
    at Object.createClient (/opt/Mapeo/resources/app.asar/node_modules/electron-updater/out/providerFactory.js:43:20)
    at AppImageUpdater.setFeedURL (/opt/Mapeo/resources/app.asar/node_modules/electron-updater/out/AppUpdater.js:165:42)
    at new MapeoUpdater (/opt/Mapeo/resources/app.asar/src/main/auto-updater.js:31:17)
    at Object.<anonymous> (/opt/Mapeo/resources/app.asar/src/main/auto-updater.js:192:25)
    at Module._compile (internal/modules/cjs/loader.js:967:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1004:10)



```

**Comment by arky** [comment]
Thanks @achou11  

Tested the build. Confirm that https://github.com/digidem/mapeo-desktop/pull/501 build artifacts seems to resolve this issue.

**Comment by achou11** [comment]
Fixed by #501
